Add FREE attribute to some inherently free encounters

Malibu-Stacey opened the pull request:
As title.

midgleyc commented:
Thanks!

You've not quite got the syntax right -- it's a space, not a tab, after free -- but I'll fix it up and add a couple more omitted free monsters.

midgleyc commented on src/data/monsters.txt:
```diff
@@ -2631,7 +2631,7 @@ Cinco de Mayo reveler	1943	reveler1.gif,reveler2.gif	Scale: 5 Init: -10000 Meat:
 # Eldritch Incursion (October 2016)
 
-Eldritch Tentacle	1974	eldtentacle.gif	Scale: 7 Cap: 400 Floor: 20 Init: -10000 P: horror Phys: 50 Elem: 50 E: spooky Article: an	eldritch effluvium (0)	eldritch effluvium (0)	eldritch effluvium (0)
+Eldritch Tentacle	1974	eldtentacle.gif	FREE	Scale: 7 Cap: 400 Floor: 20 Init: -10000 P: horror Phys: 50 Elem: 50 E: spooky Article: an	eldritch effluvium (0)	eldritch effluvium (0)	eldritch effluvium (0)
```
```suggestion
Eldritch Tentacle	1974	eldtentacle.gif	FREE Scale: 7 Cap: 400 Floor: 20 Init: -10000 P: horror Phys: 50 Elem: 50 E: spooky Article: an	eldritch effluvium (0)	eldritch effluvium (0)	eldritch effluvium (0)
```

midgleyc reviewed (changes requested):
Okay, so I can't fix it because I don't have permissions to push to your repo. But if you could make the following monsters free:
* lynyrd
* %monster%
* the Tunnel of L.O.V.E. enemies
* the Cargo Cultist Shorts bell fights
* Sssshhsssblllrrggghsssssggggrrgglsssshhssslblgl

That would be nice.

Autoscend also says that Travoltron, the Peripatetic Pasta and the wandering accordionists are free, so maybe those too?

Malibu-Stacey commented:
> Okay, so I can't fix it because I don't have permissions to push to your repo. But if you could make the following monsters free:

That's bizarre, there's no permission changes since the last PR.

> * lynyrd
> * %monster%
> * the Tunnel of L.O.V.E. enemies
> * the Cargo Cultist Shorts bell fights
> * Sssshhsssblllrrggghsssssggggrrgglsssshhssslblgl
> 
> That would be nice.

Can do!

> Autoscend also says that Travoltron, the Peripatetic Pasta and the wandering accordionists are free, so maybe those too?

I think it's wrong about Travoltron but I'll double check in the game itself after rollover when the account I use to test autoscend is done with its current run. The others are ancient revamp wanderers which I don't think there's any way to fight now anyway.

midgleyc commented on src/data/monsters.txt:
```diff
@@ -401,7 +401,7 @@ lowercase K	938	lower_k.gif	Atk: 57 Def: 49 HP: 60 Init: 45 P: humanoid Article:
 lumberjill	234	lumberjill.gif	Atk: 36 Def: 32 HP: 36 Init: 60 Meat: 20 P: dude Article: a	Blatantly Canadian (30)	maple syrup (30)	supportive bra (c20)
 lumberjuan	235	lumberjuan.gif	Atk: 35 Def: 31 HP: 35 Init: 60 Meat: 20 P: dude Article: a	los chinos (5)	leaflet (15)	wooden axe (5)
-lynyrd	1533	lynyrd.gif	Atk: 150 Def: 150 HP: 155 Init: 10000 P: beast Article: a	lynyrd skin (100)
+lynyrd	1533	FREE lynyrd.gif	Atk: 150 Def: 150 HP: 155 Init: 10000 P: beast Article: a	lynyrd skin (100)
```
```suggestion
lynyrd	1533	lynyrd.gif	FREE Atk: 150 Def: 150 HP: 155 Init: 10000 P: beast Article: a	lynyrd skin (100)
```

Malibu-Stacey commented:
Confirming Travoltron costs an adventure to kill. autoscend is definitely wrong about that one. At 15 advs before the fight I cleavered it to death in the relay browser and had 14 advs afterwards and no message in the GCLI of "This combat did not cost a turn".

Malibu-Stacey commented:
> It might be because this one is from your `main` branch?

The CBB consumables one was on my main branch too. I haven't created any branches in my fork ever. I'm blaming GitHub for being weird.

midgleyc reviewed (approved):
Thanks!
